src/test/java/core/basesyntax/exception/InsufficientQuantityExceptionTest.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.exception;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class InsufficientQuantityExceptionTest {
+    @Test
+    void exception_printCorrectMessage_ok() {

```

Comment:
This test is not really checking anything, it's better to test the exception where it can be thrown.